### PR TITLE
Readme: a better positioning with ML loggers

### PR DESCRIPTION
**Pull request by dmpetrov**

### CODE REVIEW

**Review by pared — APPROVED** [review]

**Comment by dberenbaum** [comment]
Should Jorge review for style?

**Comment by dmpetrov** [comment]
@jorgeorpinel I'd appreciate it if you can take a look.
No goals to make it perfect - just an improvement.

**Comment by shcheklein** [comment]
@dmpetrov @dberenbaum I suggest that we merge this and we can do the style review asynchronously after (even editing file directly). This is early stage, not the final pass on this README, it's fine to move faster in this case I think. WDYT?

**Comment by dberenbaum** [comment]
Sure, no need to hold things up.

File: README.rst [hunk 1]
```diff
@@ -1,16 +1,32 @@
...
+
+
+Python API
+==========
+
+DVCLive is a Python library. The interface consists of three main methods:
 
 1. :code:`dvclive.init(path)` - initializes dvclive logger. The metrics will be saved under :code:`path`.
 
 2. :code:`dvclive.log(metric, value, step)` - logs the metric value. The value and step will be appended to :code:`path/{metric}.tsv` file. The step value is optional.
```
Comment (jorgeorpinel) [inline README.rst:24]:
```suggestion
2. :code:`dvclive.log(metric, value, step)` - logs the metric value. The :code:`value` and :code:`step` (optional) will be appended to :code:`path/{metric}.tsv` file.
```

File: README.rst [hunk 2]
```diff
@@ -1,16 +1,32 @@
...
 
 3. :code:`dvclive.next_step()` - signals :code:`dvclive` that current step has ended. Executed automatically if same :code:`metric` is logged again.
```
Comment (jorgeorpinel) [inline README.rst:25]:
```suggestion
3. :code:`dvclive.next_step()` - signals that the current step has ended (implied when the same :code:`metric` is logged again).
```

File: README.rst [hunk 3]
```diff
@@ -1,16 +1,32 @@
...
+
+
+Call to collaboration
+=====================
+
+Today only Python language is supported (while DVC is language agnostic) with a minimum number of connectors to libs (Keras, XGBoost).
+DVC team is happy to help in extending the functionality - please `create an issue <https://github.com/iterative/dvclive/issues>`_ to start a discussion.
```
Comment (jorgeorpinel) [inline README.rst:32]:
```suggestion
Today only Python is supported (while DVC is language agnostic), with a minimum number of connectors to ML libs (Keras, XGBoost).
The DVCLive team is happy to extend the functionality as needed. Please `create an issue <https://github.com/iterative/dvclive/issues>`_ to start a discussion!
```

**Review by jorgeorpinel — COMMENTED** [review]
Just copy edits

**Comment by dmpetrov** [comment]
thank you @jorgeorpinel ! I applied all the suggestions and merged it.

**Pull request merged**